lib/mocha.js

```diff
@@ -171,6 +174,19 @@ function Mocha(options) {
   if (options.rootHooks) {
     this.rootHooks(options.rootHooks);
   }
+  if (options.parallel && options.jobs > 1) {
```

Comment:
Yes, but probably a bad decision.  The reason was "I hate the API"

Reply:
OK, the reason was that changing the value after/during a call to `run()` is not going to work.  This is why I limited it to being set by the constructor... though it might be necessary to instantiate a `Mocha` and _then_ enable parallel mode, but I'm not sure of the use case.

At any rate, I'm adding a fluent setter fn which handles this, but it needs to make extra checks to decide whether it's safe to enable.

Reply:
I personally feel like setting `parallel` _after_ a run should be fine. Even during it should be fine, as you would be setting it for the next run.